### DISCUSSION TITLE
Microsoft Update LTS Upgrade Annoucement

### DISCUSSION THREAD

**Discussion by StevenBucher98** [PowerShell/PowerShell#24328] [General]
Since PowerShell 7.2 goes out of support on November 8th 2024, after this happens, we will be automatically upgrading folks on 7.2.x to 7.4.x (latest) who are using Microsoft Update. This discussion serves as an announcement for this behavior. You can find more details about MU behavior in our [MU FAQ](https://learn.microsoft.com/en-us/powershell/scripting/install/microsoft-update-faq?view=powershell-7.4).

**Comment by rhubarb-geek-nz** [comment]
This also applies to [PowerShell 7.2 Azure Functions](https://azure.microsoft.com/en-us/updates/v2/powershell72-azure-functions-retirement#:~:text=Azure%20Functions-,Support%20for%20PowerShell%207.2%20on%20Azure%20Functions%20is%20ending%20on,Azure%20Functions%20stack%20version%20support.) (not mentioned in above FAQ)
